app/src/hooks/AuthHandler.ts

```diff
@@ -117,6 +127,8 @@ const AuthHandler: React.FC<{}> = () => {
         window.isSyncEnabled = isSyncEnabled;
         window.keySetDoneisSyncEnabled = true;
 
+        const mailType = await getEmailType(user.email);
```

Comment:
Can we add this call to `getEmailType` in the Promise.all interables in line 117